roles/candlepin/templates/candlepin.conf.j2

```diff
@@ -9,7 +9,7 @@ candlepin.audit.hornetq.config_path=/etc/candlepin/broker.xml
 candlepin.db.database_manage_on_startup=Manage
 
 candlepin.auth.oauth.enable=true
-candlepin.auth.oauth.consumer.katello.secret={{ candlepin_oauth_secret }}
+candlepin.auth.oauth.consumer.katello.secret="NOTTHEREALSECERT"
```

Comment:
Do we even need to include this in the config file if we're not passing it? Right now I can see a failure mode where it somehow isn't provided/respected and the password ends up being `NOTTHEREALSECERT`. If you don't provide at all I'd hope that Candlepin has a sane default that refuses to work.

Reply:
I have it here to test whether the environment variables overwrite what's in the config. To see if that provides a way to ship defaults but know that we can provide a configuration option to users if need be by adding the environment variable.

If we went this route as a whole, the question would be do we include this in the `.conf` file available in the container. I could see for anything that is a "secret" we opt *not* to include it. For other elements, we include a sane default that can be overridden.